### DISCUSSION TITLE
Csla 6.xx and .NET 6.0

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#2845] [Questions] [answered]
We have a project in the pipeline and would like to get a head start. We will be using Blazor and Xamarin (?) UIs.
I downloaded Csla Samples and they seem to target .Net standard and Core. 
Since we have a couple of months (at least), is it too ambitious to start thinking about targeting .NET 6.0? 

Kind regards

**Comment by rockfordlhotka** [comment, marked as answer]
I hope it isn't too ambitious! I think we are very close to the end of the CSLA 6 release, which is what you'll probably want.

CSLA 5 works great with Xamarin and Blazor wasm. CSLA 6 works (correctly) with server-side Blazor. No real changes for Xamarin, other than the broader set of (big) changes in CSLA 6 that affect everyone.

**Reply by Chicagoan2016** [reply]
Thank you @rockfordlhotka , so with Csla 6 we will be targeting .NET 6 (only)?
Regards

**Reply by michaelcsikos** [reply]
CSLA 6 will still target:
- .NET Framework 4.6.2, 4.7.2, 4.8
- .NET Standard 2.0
- .NET ~~5.0~~, 6.0

**Reply by michaelcsikos** [reply]
However, if you are using Blazor Server skip .NET 5.0 and only target .NET 6.0+

**Reply by rockfordlhotka** [reply]
@michaelcsikos I'm actually going to drop .NET 5 from the CSLA 6 release, since .NET 5 goes off support from Microsoft in just a few weeks. Otherwise your list is correct.
